assets/styles/css/styles.css

```diff
@@ -97,234 +196,483 @@ button {
+  }
+  .campaigns__item:nth-child(3) {
+    padding: 0 0 0 28px;
+    align-self: flex-start;
+  }
+}
 
 .campaigns__content {
-  width: 320px;
-  height: 196px;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.campaigns__content:last-child {
+}
+
+.campaigns__item:last-child .campaigns__body {
+  padding: 0;
 }
 
-.campaigns__content > a {
-  padding-left: 30px;
+.campaigns__body a:last-child {
   font-weight: 800;
   font-size: 16px;
-  line-height: 19px;
+  line-height: 119%;
   color: #4ba87d;
 }
+.campaigns__body > a:hover {
+  text-decoration: underline;
+}
+@media screen and (max-width: 768px) {
+  .campaigns__content a:last-child {
+    font-size: 12px;
+    line-height: 14px;
+  }
+}
 
 .campaigns__thumbnail {
-  width: 320px;
+  width: 100%;
+  max-width: 320px;
 }
 
 .campaigns__image {
-  width: 320px;
-  height: 260px;
+  flex: 0 0 56%;
+  display: block;
+  width: 100%;
   background-size: cover;
+  background-position: center;
 }
 
-.campaigns__image_first {
-  background-image: url("/assets/images/learnMore/learn-more-1.jpg");
+@media screen and (max-width: 768px) {
+  .campaigns__image {
+    height: 166px;
+    width: 209px;
+  }
 }
 
-.campaigns__image_second {
-  background-image: url("/assets/images/learnMore/learn-more-2.jpg");
+.campaigns__body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 0 20px;
+  height: 100%;
+  font-size: 16px;
+  line-height: 26px;
+}
+@media screen and (max-width: 768px) {
+  .campaigns__body {
+    font-size: 12px;
+    line-height: 16px;
+  }
 }
 
-.item-content {
-  padding: 24px 39px 20px 30px;
+.campaigns__body a {
+  font-size: 16px;
+  line-height: 162%;
 }
 
 .campaigns__header {
   padding-top: 50px;
-  font-family: "Playfair Display";
-  font-weight: 900;
-  font-size: 44px;
-  line-height: 48px;
-  color: #000000;
+  text-align: inherit;
 }
-
-.campaigns__text-content {
-  padding-top: 24px;
-}
-
-.campaigns__item_text {
-  align-self: flex-end;
-  box-shadow: none;
-  background: none;
+@media screen and (max-width: 768px) {
+  .campaigns__header {
+    font-size: 28px;
+    line-height: 31px;
+  }
 }
 
 .learn__images {
   display: flex;
   justify-content: center;
   align-items: center;
+  margin: 133px 0 40px 0;
+}
+@media screen and (max-width: 1100px) {
+  .learn__images {
+    margin: 0;
+    padding: 0 0 60px 0;
+    flex-wrap: wrap;
+  }
 }
 
 .learn__image {
-  position: relative;
-  padding: 16px 16px 40px 16px;
-  height: 100%;
-  width: 100%;
+  flex: 0 0 auto;
+  margin: 10px;
+  width: 342px;
+  height: 239px;
   color: #ffffff;
 }
+@media screen and (max-width: 780px) {
+  .learn__image {
+    width: 223px;
+    height: 157px;
+  }
+}
 
 .learn__image > div {
-  width: 342px;
-  height: 239px;
-  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+}
+
+.card {
+  display: flex;
+  flex-direction: column-reverse;
 }
 
 .learn__first {
+  background-position: center;
+  background-size: cover;
```

Comment:
in case you have something like that

```
<div className="container">
  <div className="element-1"></div>
  <div className="element-2"></div>
  <div className="element-3"></div>
</div>
```

instead of attaching same styles for each `element-X` you can define common selector for container elements.

```
(in case we have not deep nesting of div elements)
.container > div {
// some styles
}

or

.container .element {
// some styles
}
```